examples/markstream-react-example/src/components/Animation/slicer.scss

```
@@ -0,0 +1,71 @@
+
+// Please change your favorite font-size!
+$fontSize: 8rem;
+
+.slicer-gradient {
+  .text {
+    position: absolute;
+    font-size: $fontSize;
+    color: #fff;
+    line-height: $fontSize;
+    opacity: 0;
+    font-family: 'Anton', sans-serif;
```

Stylelint (17.14.1)

[error] Expected no quotes around "Anton" (font-family-name-quotes)

(font-family-name-quotes)